### DISCUSSION TITLE
Conditional Free Shipping Applies Incorrectly Across Multiple Vendors

### DISCUSSION THREAD

**Discussion by Fayeakuzzamantonmoy** [getdokan/dokan#2488] [Bug / Problem]
**Description:**  
When using WooCommerce's conditional free shipping option with the Dokan plugin, the shipping rules are not being applied as expected in multi-vendor scenarios.  

### Steps to Reproduce:  
1. Enable conditional free shipping in WooCommerce (e.g., **Free shipping for orders with a minimum amount of $X**).  
2. Set a minimum order amount for free shipping (e.g., $50).  
3. Place an order from two different vendors:  
   - Vendor A's order meets the minimum order amount for free shipping.  
   - Vendor B's order does not meet the minimum order amount for free shipping.  
4. Proceed to checkout.  

### Expected Behavior:  
Free shipping should apply **only to Vendor A's products** that fulfill the minimum order amount requirement. Vendor B's products should have the standard shipping charges applied.  

### Actual Behavior:  
Free shipping is applied to **all vendors' products**, regardless of whether the individual vendor's order meets the minimum order amount requirement.  

### Impact:  
This behavior leads to incorrect shipping calculations and could result in lost revenue for vendors who do not meet the free shipping criteria.  

### Environment Details:  
- **Dokan Plugin Version:** 3.14.3  
- **WooCommerce Version:** 9.5.1  

### Screenshots/References:  
Shipping Conditions - https://snipboard.io/cHehmX.jpg
Cart Page - https://snipboard.io/bt27Ve.jpg 

### CC:  
@jamil-mahmud Bhai

**Comment by jamil-mahmud** [comment]
Further discussion will continue on #2632. Closing accordingly.
